src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -1025,6 +1025,38 @@ TEST_F(MessageHelperTest,
               std::find(status_array->begin(), status_array->end(), item_2_so));
 }
 #endif
+
+TEST_F(MessageHelperTest, SendNaviSetVideoConfigRequest) {
+  smart_objects::SmartObjectSPtr result;
+  EXPECT_CALL(mock_application_manager, ManageHMICommand(_))
+      .WillOnce(DoAll(SaveArg<0>(&result), Return(true)));
+
+  int32_t app_id = 123;
+  smart_objects::SmartObject video_params(smart_objects::SmartType_Map);
+  video_params["protocol"] = hmi_apis::Common_VideoStreamingProtocol::RTP;
```

Comment:
Done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/229a2901e652be32d39e5d8559fb05f1a794dde5.